initial-ai-src/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ruter Reis Discount Calculator</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Ruter Reis Discount Calculator</h1>
+      <p>Find out what discount you can get on Oslo public transport tickets</p>
+    </header>
+
+    <main>
+      <div class="calculator-container">
+        <div class="input-section">
+          <label for="trips-per-week"
+            >How many trips do you expect per week?</label
+          >
+          <input
+            type="number"
+            id="trips-per-week"
+            min="0"
+            max="50"
+            value="10"
```

Comment:
❓ What's the justification for a default of 10? Just curious, seems reasonable if someone only uses public transport on weekdays to go to/from the office.